### PR TITLE
fix: [#177161429] Fixed missing loading on delete credit card

### PR DESCRIPTION
**Pull request by andrea-favaro**
## Short description
Show loading when canceling a credit card.

**After clicking this delete button**
Screenshot 2021-04-13 at 14 50 33

**A loading page like this will be shown**
Screenshot 2021-04-13 at 14 56 01

## How to test
Move into credit card detail and try to delete it. I suggest you to add a delay in dev server to slow down the deletion process and be able to properly observe the loading behavior

### CODE REVIEW

File: ts/screens/wallet/TransactionsScreen.tsx [hunk 1]
```diff
@@ -161,6 +167,17 @@ const TransactionsScreen: React.FC<Props> = (props: Props) => {
...
+    return (
+      <LoadingErrorComponent
+        isLoading={props.isLoadingDelete}
+        loadingCaption={loadingCaption}
+        onRetry={onRetry}
+      />
+    );
+  }
+
   return (
```
Comment (debiff) [inline ts/screens/wallet/TransactionsScreen.tsx:181]: Can this reduce your code size?
```suggestion
  return props.isLoadingDelete ? (
    <LoadingErrorComponent
      isLoading={props.isLoadingDelete}
      loadingCaption={I18n.t("cardComponent.deleteLoading")}
      onRetry={onRetry}
    />
  ) : (
```

**Review by debiff — CHANGES_REQUESTED** [review]

File: ts/features/wallet/bancomat/screen/BancomatDetailScreen.tsx [hunk 2]
```diff
@@ -82,37 +85,50 @@ const BancomatDetailScreen: React.FunctionComponent<Props> = props => {
...
-        <ItemSeparatorComponent noPadded={true} />
-        <View spacer={true} />
-        <PaymentMethodCapabilities paymentMethod={bancomat} />
-        <View spacer={true} />
-        <ItemSeparatorComponent noPadded={true} />
-        <View spacer={true} />
-        <UnsubscribeButton
-          onPress={() => present(() => props.deleteWallet(bancomat.idWallet))}
+    <>
+      {props.isLoadingDelete ? (
```
Comment (debiff) [inline ts/features/wallet/bancomat/screen/BancomatDetailScreen.tsx:89]: There is a reason why this fragment is used? Can we remove it in your opinion?
```suggestion
    props.isLoadingDelete ? (
```
If for you make sense the same is in the other payment methods screen.

File: ts/features/wallet/bancomat/screen/BancomatDetailScreen.tsx [hunk 3]
```diff
@@ -82,37 +85,50 @@ const BancomatDetailScreen: React.FunctionComponent<Props> = props => {
...
+            <UnsubscribeButton
+              onPress={() =>
+                present(() => props.deleteWallet(bancomat.idWallet))
+              }
+            />
+          </View>
+          <View spacer={true} extralarge={true} />
+        </DarkLayout>
+      )}
+    </>
```
Comment (debiff) [inline ts/features/wallet/bancomat/screen/BancomatDetailScreen.tsx:131]: Connected to the above comment:
```suggestion
      )
```

**Review by debiff — CHANGES_REQUESTED** [review]

**Review by debiff — APPROVED** [review]
This PR lgtm. 
@fabriziofff can you double check it? 
@andrea-favaro can you solve the conflicts please?

**Comment by fabriziofff** [comment]
@debiff sure, I wait for the conflict resolution and do a complete check.

**Comment by andrea-favaro** [comment]
> This PR lgtm.
> @fabriziofff can you double check it?
> @andrea-favaro can you solve the conflicts please?

@fabriziofff conflict solved

**Review by fabriziofff — APPROVED** [review]

**Pull request merged**